oshi-core/src/main/java/oshi/hardware/HWPartition.java

```diff
@@ -350,4 +351,17 @@ public boolean equals(Object obj) {
         return true;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(" |-- ").append(getIdentification()).append(": ");
```

Comment:
Similar to earlier comment let's keep the indentation (`" |-- "`) in the `SystemInfoTest` output and start the toString with text.